src/main/java/com/jwt/mybatis/jwtApplication/security/JwtAuthenticationEntryPoint.java

```diff
@@ -0,0 +1,21 @@
+package com.jwt.mybatis.jwtApplication.security;
+
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.web.AuthenticationEntryPoint;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@Component
+public class JwtAuthenticationEntryPoint implements AuthenticationEntryPoint {
+    @Override
+    public void commence(HttpServletRequest request, HttpServletResponse response, AuthenticationException authException) throws IOException {
```

Comment:
Since we are just generating token from this app, do we need this?
What is the use of this?

Reply:
No, When we try to access a secured endpoint without providing valid jwt token then this will get invoked with custom message. 
for testing jwt added this. Removed now